src/components/input/_input.scss

```diff
@@ -34,7 +34,7 @@
   }
 
   .govuk-c-input--error {
-    border: $govuk-border-width-error solid $govuk-error-colour;
+    border: $govuk-border-width-form-element-error solid $govuk-red;
```

Comment:
Should we be using $govuk-error-colour here not $red?

Reply:
Yes, good catch.